lib/app_web/live/app_live.html.heex

```diff
@@ -6,13 +6,22 @@
     <textarea 
       class="w-full py-1 px-1 text-slate-800 text-3xl
         bg-white bg-clip-padding
+        resize-none
+        max-h-80
```

Comment:
see https://tailwindcss.com/docs/max-height

I've also added `resize-none` to disable the manually resize UI